west.yml

```diff
@@ -301,7 +301,7 @@ manifest:
       revision: da78aea63159771956fe0c9263f2e6985b66e9d5
       path: modules/lib/open-amp
     - name: openthread
-      revision: 7761b81d23b10b3d5ee21b8504c67535cde10896
+      revision: pull/140/head
```

Comment:
Please update SHA: `49c59ec519cc8b49dd58978d1bc80b7ae7ba88d0`

Reply:
updated and rebased